5-infrastructure/modules/base-env/boa-gke.tf

```
@@ -57,6 +57,8 @@ locals {
       master_authorized_networks = []
     }
   }
+  bin_auth_attestors = [for attestor in var.bin_auth_attestor_names : "projects/${var.bin_auth_attetor_project_id}/attestors/${attestor}"]
+  whitelist_patterns = ["quay.io/random-containers/*", "k8s.gcr.io/more-random/*", "gcr.io/${var.boa_gke_project_id}/*"] # Example
```

lets use allow/deny lists here and throughout
```suggestion
  allowlist_patterns = ["quay.io/random-containers/*", "k8s.gcr.io/more-random/*", "gcr.io/${var.boa_gke_project_id}/*"] # Example
```